lib/internal/abort_controller.js

```diff
@@ -312,6 +365,10 @@ function abortSignal(signal, reason) {
     [kTrustEvent]: true,
   });
   signal.dispatchEvent(event);
+  signal[kDependantSignals]?.forEach((s) => {
```

Comment:
Should use `ArrayPrototypeForEach` from `primordials`?

Reply:
`signal[kDependantSignals]` is a `SafeSet` from `primordials`, so I thought we could use `forEach` directly. is it better to use `SetPrototypeForEach`?

Reply:
You can and it's safe